### DISCUSSION TITLE
PICKCHARS not available in 2.6.6 and 2.7.0 won't start

### DISCUSSION THREAD

**Discussion by Anne-Marie2021** [keepassxreboot/keepassxc#6883] [Q&A] [answered]
Hello fellow GitHubers!

I am asking you for help with PICKCHARS (https://keepass.info/help/base/placeholders.html#pickchars).

Yesterday I used Windows & KeePass Password Safe and today I have started using Mac OS & KeePassXC. 

I am too much of a layman and did not succed in bypassing KPPS with Mono. Unfortunately, **neither of these four** **in KeePassXC 2.6.6** ({PICKCHARS}, {PICKCHARS}{ENTER}, {PICKCHARS:password}, {PICKCHARS}{PASSWORD}) **show a dialog** in which I can pick characters of my password I need for my bank account (https://login.ingbank.pl/mojeing/app/#login). 

Mr droidmonkey said on 3 December 2020 that **PICKCHARS was going to be merged into KeePassXC 2.7.0** (https://github.com/keepassxreboot/keepassxc/issues/725). I found 10 builds of 2.7.0 from 31/07/2021 to 30/08/2021 (https://snapshot.keepassxc.org/). Unfortunately, they will not start. You can find the bug report below.

[Report.pdf](https://github.com/keepassxreboot/keepassxc/files/7091117/Report.pdf)

1. When should we expect 2.7.0 version with PICKCHARS?
2. Is there a way for me to run 2.7.0 snapshot in order to use PICKCHARS today?

Thank you in advance and I wish you good health!
Anne-Marie2021

**Comment by droidmonkey** [comment, marked as answer]
If you are on ARM mac then you'll need to build 2.7.0 on your own. We don't have a release date yet.

**Reply by Anne-Marie2021** [reply]
Thank you for your answer. I am on Intel's x-86-64 (Macbook mid 2010 with Intel Core 2 Duo).

**Reply by droidmonkey** [reply]
Then the snapshot dmg should work fine. You might have to explicitly allow it to run since it is unsigned

**Reply by Anne-Marie2021** [reply]
I executed "sudo spctl --master-disable" in Terminal, reinstalled snapshot dmg, rebooted MacBook and opened it while holding CTRL. 
The situation remains the same and I am still getting the bug report.

**Reply by droidmonkey** [reply]
Hmm that's interesting you are crashing in the botan library. This obviously doesn't occur in our test server since the snapshot would never be created in the first place.
